docs/SUPPORT_MATRIX.md

```
@@ -16,12 +16,13 @@
 | 21 | get_process_name | ✅ Full | |
 | 22 | yield | ✅ Full | |
 | 30 | map_memory | ✅ Full | |
-| 31 | unmap_memory | ✅ Full | |
-| 32 | protect_memory | ⚠️ Stub | |
-| 40 | create_event | ✅ Full | |
+|| 31 | unmap_memory | ✅ Full | |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 4; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)